From af7f7989eff56cae1c7440a9749a501af681ba8d Mon Sep 17 00:00:00 2001
From: Craig Kewley <craigx.kewley@intel.com>
Date: Fri, 7 Apr 2017 08:30:38 +0100
Subject: [PATCH 065/102] ASoC: Intel: Boards: remove ignore_suspend flag

If the ignore_suspend flag is set then the dpib and lpib positions
won't be saved and the DMA won't be resumed. Set the flags to 0 for
all DAI links.

Signed-off-by: Craig Kewley <craigx.kewley@intel.com>
---
 sound/soc/intel/boards/bxt_gpmrb.c | 16 +---------------
 1 file changed, 1 insertion(+), 15 deletions(-)

diff --git a/sound/soc/intel/boards/bxt_gpmrb.c b/sound/soc/intel/boards/bxt_gpmrb.c
index ec80cb0..863f1e9 100644
--- a/sound/soc/intel/boards/bxt_gpmrb.c
+++ b/sound/soc/intel/boards/bxt_gpmrb.c
@@ -268,7 +268,6 @@ static int broxton_gpmrb_tdf8532_startup(struct snd_pcm_substream *substream)
 		.platform_name = "0000:00:0e.0",
 		.init = NULL,
 		.dpcm_capture = 1,
-		.ignore_suspend = 1,
 		.nonatomic = 1,
 		.dynamic = 1,
 		.ops = &broxton_gpmrb_dirana_ops,
@@ -282,7 +281,6 @@ static int broxton_gpmrb_tdf8532_startup(struct snd_pcm_substream *substream)
 		.platform_name = "0000:00:0e.0",
 		.init = NULL,
 		.dpcm_capture = 1,
-		.ignore_suspend = 1,
 		.nonatomic = 1,
 		.dynamic = 1,
 		.ops = &broxton_gpmrb_dirana_ops,
@@ -296,7 +294,6 @@ static int broxton_gpmrb_tdf8532_startup(struct snd_pcm_substream *substream)
 		.platform_name = "0000:00:0e.0",
 		.init = NULL,
 		.dpcm_capture = 1,
-		.ignore_suspend = 1,
 		.nonatomic = 1,
 		.dynamic = 1,
 		.ops = &broxton_gpmrb_dirana_ops,
@@ -324,7 +321,6 @@ static int broxton_gpmrb_tdf8532_startup(struct snd_pcm_substream *substream)
 		.platform_name = "0000:00:0e.0",
 		.init = NULL,
 		.dpcm_capture = 1,
-		.ignore_suspend = 1,
 		.nonatomic = 1,
 		.dynamic = 1,
 	},
@@ -350,7 +346,6 @@ static int broxton_gpmrb_tdf8532_startup(struct snd_pcm_substream *substream)
 		.platform_name = "0000:00:0e.0",
 		.init = NULL,
 		.dpcm_capture = 1,
-		.ignore_suspend = 1,
 		.nonatomic = 1,
 		.dynamic = 1,
 		.ops = &broxton_gpmrb_bt_modem_ops,
@@ -378,7 +373,6 @@ static int broxton_gpmrb_tdf8532_startup(struct snd_pcm_substream *substream)
 		.platform_name = "0000:00:0e.0",
 		.init = NULL,
 		.dpcm_capture = 1,
-		.ignore_suspend = 1,
 		.nonatomic = 1,
 		.dynamic = 1,
 		.ops = &broxton_gpmrb_bt_modem_ops,
@@ -406,7 +400,6 @@ static int broxton_gpmrb_tdf8532_startup(struct snd_pcm_substream *substream)
 		.platform_name = "0000:00:0e.0",
 		.init = NULL,
 		.dpcm_capture = 1,
-		.ignore_suspend = 1,
 		.nonatomic = 1,
 		.dynamic = 1,
 		.ops = &broxton_gpmrb_hdmi_ops,
@@ -420,7 +413,6 @@ static int broxton_gpmrb_tdf8532_startup(struct snd_pcm_substream *substream)
 		.codec_dai_name = "snd-soc-dummy-dai",
 		.platform_name = "0000:00:0e.0",
 		.capture_only = true,
-		.ignore_suspend = 1,
 	},
 	{
 		.name = "Bxt Trace Buffer1",
@@ -430,7 +422,6 @@ static int broxton_gpmrb_tdf8532_startup(struct snd_pcm_substream *substream)
 		.codec_dai_name = "snd-soc-dummy-dai",
 		.platform_name = "0000:00:0e.0",
 		.capture_only = true,
-		.ignore_suspend = 1,
 	},
 	{
 		.name = "Bxt Compress Probe playback",
@@ -461,7 +452,6 @@ static int broxton_gpmrb_tdf8532_startup(struct snd_pcm_substream *substream)
 		.codec_name = "snd-soc-dummy",
 		.codec_dai_name = "snd-soc-dummy-dai",
 		.platform_name = "0000:00:0e.0",
-		.ignore_suspend = 1,
 		.dpcm_capture = 1,
 		.dpcm_playback = 1,
 		.no_pcm = 1,
@@ -474,7 +464,6 @@ static int broxton_gpmrb_tdf8532_startup(struct snd_pcm_substream *substream)
 		.codec_name = "snd-soc-dummy",
 		.codec_dai_name = "snd-soc-dummy-dai",
 		.platform_name = "0000:00:0e.0",
-		.ignore_suspend = 1,
 		.dpcm_capture = 1,
 		.no_pcm = 1,
 	},
@@ -486,7 +475,6 @@ static int broxton_gpmrb_tdf8532_startup(struct snd_pcm_substream *substream)
 		.codec_name = "snd-soc-dummy",
 		.codec_dai_name = "snd-soc-dummy-dai",
 		.platform_name = "0000:00:0e.0",
-		.ignore_suspend = 1,
 		.dpcm_capture = 1,
 		.dpcm_playback = 1,
 		.no_pcm = 1,
@@ -499,7 +487,6 @@ static int broxton_gpmrb_tdf8532_startup(struct snd_pcm_substream *substream)
 		.codec_name = "snd-soc-dummy",
 		.codec_dai_name = "snd-soc-dummy-dai",
 		.platform_name = "0000:00:0e.0",
-		.ignore_suspend = 1,
 		.dpcm_capture = 1,
 		.dpcm_playback = 1,
 		.no_pcm = 1,
@@ -512,7 +499,6 @@ static int broxton_gpmrb_tdf8532_startup(struct snd_pcm_substream *substream)
 		.codec_name = "i2c-INT34C3:00",
 		.codec_dai_name = "tdf8532-hifi",
 		.platform_name = "0000:00:0e.0",
-		.ignore_suspend = 1,
 		.dpcm_playback = 1,
 		.no_pcm = 1,
 	},
@@ -524,7 +510,6 @@ static int broxton_gpmrb_tdf8532_startup(struct snd_pcm_substream *substream)
 		.codec_name = "snd-soc-dummy",
 		.codec_dai_name = "snd-soc-dummy-dai",
 		.platform_name = "0000:00:0e.0",
-		.ignore_suspend = 1,
 		.dpcm_capture = 1,
 		.dpcm_playback = 1,
 		.no_pcm = 1,
@@ -579,6 +564,7 @@ static int broxton_audio_remove(struct platform_device *pdev)
 	.remove = broxton_audio_remove,
 	.driver = {
 		.name = "gpmrb_machine",
+		.pm = &snd_soc_pm_ops,
 	},
 };
 
-- 
1.9.1

